src/router/router.tsx

```diff
@@ -8,7 +8,60 @@ const router = createBrowserRouter([
   },
   {
     path: '/how-to-play',
-    element: <HomePage />,
+    children: [
+      {
+        path: 'overview',
+        element: <h1>How to play</h1>,
+      },
+      {
+        path: 'hosts-1',
```

Comment:
This will make a little more sense later on when we use the `<Outlet />` component for common layouts.

https://reactrouter.com/en/main/components/outlet

Reply:
For now, you can just use `<Outlet />` as the 'hosts' element, and a `<div>` for the `1` path